beers/models.py

```diff
@@ -80,3 +115,58 @@ class Manufacturer(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Beer(models.Model):
+    name = models.CharField(max_length=25, db_index=True)
```

Comment:
Now that I think about it, I actually think that 25 could be too short.  Especially given the crazy names as well as collaboration beers that are out there now.

Reply:
I'm good with extending to 50.

Reply:
So maybe Unicode as well?

* Stone Enjoy By 01.01.19 Brüt IPA

And finding longer beer names: 

* Cigar City  => Marshal Zhukov's - Apple Brandy Barrel Aged W/ Madagascar Vanilla Beans (71)

Reply:
Unicode is already taken care of for free.

Reply:
Extended to 100 characters